src/main/java/ai/lucidtech/las/sdk/Client.java

```diff
@@ -45,315 +44,1079 @@ public Client(Credentials credentials) {
+     *
+     * @return Documents response from REST API
+     * @throws IOException General IOException
+     * @throws APIException Raised when API returns an erroneous status code
+     * @throws MissingAccessTokenException Raised if access token cannot be obtained
+     */
+    public JSONObject listDocuments() throws IOException, APIException, MissingAccessTokenException {
+        return this.listDocuments(new ListDocumentsOptions());
+    }
+
+    /**
+     * Delete documents, calls the DELETE /documents endpoint.
+     *
+     * @see Client#createDocument
+     * @return Documents response from REST API
+     * @throws IOException General IOException
+     * @throws APIException Raised when API returns an erroneous status code
+     * @throws MissingAccessTokenException Raised if access token cannot be obtained
+     */
+    public JSONObject deleteDocuments() throws IOException, APIException, MissingAccessTokenException {
+        return this.deleteDocuments(null);
+    }
+
+    /**
+     * Delete documents with the provided consentId, calls the DELETE /documents endpoint.
+     *
+     * @see Client#createDocument
+     * @param consentId Delete documents with this consentId
+     * @return Documents response from REST API
+     * @throws IOException General IOException
+     * @throws APIException Raised when API returns an erroneous status code
+     * @throws MissingAccessTokenException Raised if access token cannot be obtained
+     */
+    public JSONObject deleteDocuments(String consentId) throws IOException, APIException, MissingAccessTokenException {
+        List<NameValuePair> consent = new ArrayList<NameValuePair>();
+        if (consentId != null) {
+            consent.add(new BasicNameValuePair("consentId", consentId));
+        }
+        HttpUriRequest request = this.createAuthorizedRequest("DELETE", "/documents", consent);
+        String jsonResponse = this.executeRequest(request);
+        return new JSONObject(jsonResponse);
+    }
+
+    /**
+     *  Get document from the REST API, calls the GET /documents/{documentId} endpoint.
+     *
+     * @param documentId Id of the document
+     * @return Document response from REST API
+     * @throws IOException General IOException
+     * @throws APIException Raised when API returns an erroneous status code
+     * @throws MissingAccessTokenException Raised if access token cannot be obtained
+     */
+    public JSONObject getDocument(String documentId) throws IOException, APIException, MissingAccessTokenException {
+        HttpUriRequest request = this.createAuthorizedRequest("GET", "/documents/" + documentId);
+        String response = this.executeRequest(request);
+        return new JSONObject(response);
+    }
+
+    /**
+     * Post groundTruth to the REST API, calls the POST /documents/{documentId} endpoint.
+     * Posting groundTruth means posting the ground truth data for the particular document.
+     * This enables the API to learn from past mistakes
```

Comment:
Maybe this shouldn't be here, but specified in other documentation. This might be misleading as it does not happen necessarily